pkg/agent/apiserver/handlers/ovsflows/handler.go

```diff
@@ -87,7 +87,7 @@ func dumpMatchedGroups(aq agentquerier.AgentQuerier, groupIDs []binding.GroupIDT
 // number is invalid).
 func getTableFlows(aq agentquerier.AgentQuerier, table string) ([]Response, error) {
```

Comment:
not introduced by this PR, but do you think you could rename `table` to `tables` to match the new `getGroups`?

Reply:
Sure. Done.